WikiExtractor.py

```diff
@@ -452,11 +493,13 @@ def extract(self, out):
         self.magicWords['currentday'] = time.strftime('%d')
         self.magicWords['currenthour'] = time.strftime('%H')
         self.magicWords['currenttime'] = time.strftime('%H:%M:%S')
-        text = self.clean()
+        text = [line.encode('utf-8') for line in compact(self.clean())]
         footer = "\n</doc>\n"
+        if sum(len(line) for line in text) < Extractor.min_text_length:
```

Comment:
do we have an option for something like `-1` or the flag not being set so we can still get everything. If folks don't want this restriction it should be possible to still get everything.